mods/ts/weapons/missiles.yaml

```diff
@@ -35,9 +35,9 @@ Bazooka:
 		ImpactSound: ssplash3.aud
 		ValidImpactTypes: Water
 	Warhead@4EffAir: CreateEffect
-		Explosion: small_twlt
-		ImpactSound: expnew12.aud
-		ValidImpactTypes: Air, AirHit
+		Explosion: tiny_twlt
+		ImpactSound: expnew05.aud
+		InvalidImpactTypes: Ground
```

Comment:
Just play the TS in bleed. The air anim plays whenever it hits anything on the ground too.